django/utils/text.py

```diff
@@ -314,8 +316,27 @@ def phone2numeric(phone):
     return "".join(char2number.get(c, c) for c in phone.lower())
 
 
-def compress_string(s):
-    return gzip_compress(s, compresslevel=6, mtime=0)
+def _get_random_fname(max_random_bytes):
+    return "a" * secrets.randbelow(max_random_bytes)
+
+
+def compress_string(s, max_random_bytes=0):
```

Comment:
Make this new argument keyword-only?
```suggestion
def compress_string(s, *,  max_random_bytes=0):
```